gemd/entity/template/tests/test_process_template.py

```diff
@@ -1,9 +1,12 @@
 """Tests of the ProcessTemplate object."""
 import pytest
 
+from gemd.entity.bounds import IntegerBounds
+from gemd.entity.link_by_uid import LinkByUID
 from gemd.entity.template.process_template import ProcessTemplate
 from gemd.entity.template.condition_template import ConditionTemplate
 from gemd.entity.bounds.real_bounds import RealBounds
+from gemd.json import dumps, loads
```

Comment:
Given how you use this below, there is a `copy` method in `gemd.json` you should be using instead.
```suggestion
from gemd.json import copy
```